.eslintrc

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
Probably better to give the explicit `.json` file type since that looks like the format you chose